### PR TITLE
Handle database

### PR DESCRIPTION
**Pull request by ilyash-b**

### CODE REVIEW

**Comment by inna-svn** [comment]
@ilyash-b please review

File: src/main/java/Database.java [hunk 1]
```diff
@@ -1,27 +1,20 @@
...
     public Database() {
         try {
             connection = DriverManager.getConnection(url,"root","zubur1");
-            connection.setAutoCommit(false);
+            connection.setAutoCommit(true);
             statement = connection.createStatement();
-            connection.commit();
-        } catch (
-                SQLException sqlException) {
+        } catch (SQLException sqlException) {
```
Comment (ilyash-b) [inline src/main/java/Database.java:15]: Please either re-throw exception after `sqlException.printStackTrace();` or don't catch it. The program can not operate properly if this happens anyway.

File: src/main/java/User.java [hunk 2]
```diff
@@ -34,24 +41,36 @@ public SignInError(String message) {
...
-        //db.execution(addUser);
-        return new User(username);
+        // TODO: Validation
+
+        try {
+
+            PreparedStatement preparedStmt = Database.getDatabase().getConnection().prepareStatement("INSERT INTO users(userName,password) VALUES(?,?)");
+            preparedStmt.setString(1,username);
+            preparedStmt.setString(2,password);
+            preparedStmt.executeUpdate();
```
Comment (ilyash-b) [inline src/main/java/User.java:53]: I would assume there is a shorter way to do it. Not very important.

Comment (inna-svn) [reply src/main/java/User.java:53]: Unfortunately no shorter way to do this. it must be in separate lines

**Pull request merged**